modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/source/ClusterExtensionMessageSourceTestCase.java

```diff
@@ -53,7 +54,7 @@ public void startWhenPrimaryNode() throws Exception {
     muleContext.getNotificationManager()
         .fireNotification(new ClusterNodeNotification("you're up", PRIMARY_CLUSTER_NODE_SELECTED));
     verify(sourceAdapter, atLeastOnce()).initialise();
-    verify(sourceAdapter, times(1)).start();
+    verify(sourceAdapter, timeout(RECEIVE_TIMEOUT)).start();
```

Comment:
Shouldn't this be `timeout(RECEIVE_TIMEOUT).times(1)` to keep the original intention?